index.html

```diff
@@ -0,0 +1,314 @@
+          </a>
+          <!-- login/signup CTA -->
+
+          <!-- navigation links -->
+          <a href="/pages/wishlist-page/wishlist-page.html">
+            <span class="w-full h-full tooltip btn-icon badge-container mr-3">
+              <i
+                class="text-rose-500 text-hover-rose-300 fas fa-heart text-3xl"
+              ></i>
+              <span
+                class="icon-badge-sm bg-slate-100 text-black text-xs status-badge"
+                >1</span
+              >
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-rose-100"
+              >
+                Wish List
+              </span>
+            </span>
+          </a>
+          <a href="pages/cart-page/cart-page.html">
+            <span class="btn-icon badge-container tooltip"
+              ><i
+                class="text-rose-500 text-hover-rose-300 fas fa-shopping-cart text-3xl"
+              ></i>
+              <span
+                class="icon-badge-sm bg-rose-500 text-white text-xs status-badge bg-slate-100 text-black"
+                >2</span
+              >
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-rose-100"
+              >
+                Cart
+              </span>
+            </span>
+          </a>
+          <!-- navigation links -->
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- image slider -->
+    <section class="p-20 relative" style="top: 7rem">
+      <div class="slideshow rounded-lg overflow-hidden" style="height: 30rem">
+        <div class="slides fade">
+          <div class="slide-number">1 / 3</div>
+          <img
+            src="https://picsum.photos/id/1015/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption One</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">2 / 3</div>
+          <img
+            src="https://picsum.photos/id/1016/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption Two</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">3 / 3</div>
+          <img
+            src="https://picsum.photos/id/1039/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption Three</div>
+        </div>
+
+        <a class="prev" onclick="plusSlides(-1)">❮</a>
+        <a class="next" onclick="plusSlides(1)">❯</a>
+      </div>
+
+      <div class="mt-5" style="text-align: center">
+        <span
+          class="dot"
+          style="width: 10px; height: 10px"
+          onclick="currentSlide(1)"
+        ></span>
+        <span
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(2)"
```

Comment:
And here too, and everywhere else where you have used inline js or css both.

Reply:
I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here